### ISSUE TITLE
About your loss function and finetuning

### ISSUE THREAD

**Issue by jcliu0428** [MichaelRamamonjisoa/SharpNet#3]
Hi, I have been watching your excellent work recently. However I have not found the loss functions computation in your source code. Could you share them for us to take a look?

**Comment by MichaelRamamonjisoa** [comment]
Hi, thank you for your interest in SharpNet ! Indeed, as written in the README, I promised to add loss function computation and training code for finetuning on NYUv2 Depth. I will keep my promise, but I am a bit busy these days and had no time to clean my code. I will do so in the coming weeks, so stay tuned :)

**Comment by MichaelRamamonjisoa** [comment]
Hey, sorry for the delay. Loss code is up although it will change soon since the paper will soon be updated with a different formulation of consensus loss.
Training and evaluation code is still under cleaning. Sorry for the delay.

**Comment by MichaelRamamonjisoa** [comment]
Training code added in commit 2ede2be2117e9f5fb3075449d368ef941e643844.
